### DISCUSSION TITLE
Wifi icon to the left

### DISCUSSION THREAD

**Discussion by r57zone** [OnionUI/Onion#1594] [Ideas]
Hi, it seems that in most themes the wifi icon is superimposed on top of the battery, maybe make a slight offset to the left so that there is compatibility with non-adapted themes or just place it on the left.

**Comment by Aemiii91** [comment]
We don't have that kind of control. MainUI is closed source.

**Reply by r57zone** [reply]
It's a pity

**Comment by r57zone** [comment]
Perhaps in the binary file can find the byte responsible for offset, most likely this is the width - offset pixels, but finding the value will probably (for hex patching).
